frontend/scripts/components/profiles/chord.component.js

```diff
@@ -1,111 +0,0 @@
-import { select as d3_select } from 'd3-selection';
```

Comment:
I'm wondering if it's not best to keep this around, properly documented, in case we need chords in the future. Experience has taught us that there's definitely some love out there for chord diagrams

Reply:
☝️

Reply:
I made a separated delete commit, that shouldn't be hard to revert.

Reply:
As for documentation, it could be directly in the PR as suggested by @sorodrigo 
I'd ask @pjosh, as he's been the one writing that code (I don't think he's been deviating much from a standard D3 chord diagram though).